src/chef-mover/rebar.config

```diff
@@ -122,7 +122,6 @@
             ssh,
             uuid,
             debugger,
-            hipe,
```

Comment:
Note to reviewers: I *believe* this was only here because during the "Big Erlang Build Refactor" we hit an issue where rebar was producing wildly different build artifacts so we took a hammer to the problem and listed every single app that was in the old artifact but not in the new artifact.  Since we don't ship native builds, I don't know of a reason we would need this in the release.